### ISSUE TITLE
Add "Add Geopackage Layer" in Menu --> Layer --> Add Layer

### ISSUE THREAD

**Issue by MarcoSpaziani** [qgis/QGIS#51534]
### Feature description

Please, may you add "Add Geopackage Layer" in Menu --> Layer --> Add Layer? Thank you and regards.

**Comment by Jehan-h20220012** [comment]
Can I work on this issue? It will help me in my acedamics too

**Comment by agiudiceandrea** [comment]
@Jehan-h20220012 yes, of course. Please see https://github.com/qgis/QGIS#get-involved-with-the-community, https://qgis.org/en/site/getinvolved/development/index.html, and https://docs.qgis.org/3.22/en/docs/developers_guide/index.html.
Please notify here when you have submitted a Pull Request fixing this feature request.

**Comment by rouault** [comment]
I don't see the point in cluttering the UI for that. This is already covered by "Add a vector layer". We should rather try to reduce the number of entries in this menu

**Comment by agiudiceandrea** [comment]
> This is already covered by "Add a vector layer"

It is not the same dialog.

The "Add a vector layer" opens the "Vector" Data Source Manager dialog:
![image](https://user-images.githubusercontent.com/16253859/215406144-d3c45dc9-ff56-435b-9845-98c18ce5f0a1.png)

while the missing "Add a GeoPackage layer" should open the Data Source Manager dialog that allows to also add a GeoPackage connection:
![image](https://user-images.githubusercontent.com/16253859/215407392-4faccbcb-50d9-4716-802e-f544789fab4e.png)

I think this actually a bug, since currently there is an inconsistency between the items listed in the the "Layer -> Add Layer" menu and the items in the Data Source Manager.
Other then the GeoPackage one, only the GPS and Geonode items are also missing.

**Comment by nyalldawson** [comment]
I'm not super-keen on exposing this via the menu either -- to me it would mislead new users into thinking that it's the "correct"/"only" way to add Geopackages to their projects, when it's actually a more complex/less helpful way because it requires them to manually create a persistent connection to the file.

I'd rather they just went via the Add Vector Layer option instead, as it's much simpler to use...
